Symfony/CS/Fixer/Contrib/AlignEqualsFixer.php

```diff
@@ -24,49 +24,64 @@ class AlignEqualsFixer extends AbstractAlignFixer
     /**
      * {@inheritdoc}
      */
-    public function fix(\SplFileInfo $file, $content)
+    public function fix(\SplFileInfo $file, Tokens $tokens)
     {
-        list($tokens, $deepestLevel) = $this->injectAlignmentPlaceholders($content);
+        $this->deepestLevel = 0;
```

Comment:
I would actually prefer normalizing the other way. Having some state in the parent class which must be reset by child classes before fixing seems weird (and both child classes are actually initializing it in a different way).